controller/Session.class.php

```diff
@@ -31,6 +33,9 @@ public static function init()
             static::setNamespace(static::NAMESPACE_DEFAULT, $oldSession);
         }
 
+        $response = LBRY::logWebVisitor('lbry.io', $_SESSION[static::USER_ID], $_SERVER['REMOTE_ADDR']);
```

Comment:
1) Mildly better to be explicit about what we expect/want to be passed when `$_SESSION[static::USER_ID]` is not set.
2) lbry.io probably shouldn't be hardcoded and instead read from server

Reply:
done and done